java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -0,0 +1,37 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "HyperLogLog Commands" group for standalone and
+ * cluster clients.
+ *
+ * @see <a href="https://redis.io/commands/?group=hyperloglog">HyperLogLog Commands</a>
+ */
+public interface HyperLogLogBaseCommands {
+
+    /**
+     * Adds all elements to the HyperLogLog data structure stored at the specified <code>key</code>,
+     * creating a new structure if the <code>key</code> does not exist.
+     *
+     * <p>A command call without <code>elements</code> results in no operation being performed if the
+     * <code>key</code> already exists, or just the creation of the data structure if the <code>key
+     * </code> does not exist (in the latter case <code>1</code> is returned).
```

Comment:
```suggestion
     * </code> does not exist.
```
Returns should be discussed under `@return`.

Reply:
Fixed